### PR TITLE
Refactored styles for big screens

### LINKED ISSUE #128: Fonts fo Flat UI Design
[Services](https://active-bridge.com/services)
there're some issues with display text box on big screens. 
let's design it to be text user-friendly

- add paddings between subtitles and text boxes ( Custom web development
                                                                                 Scrum process
                                                                                 Free code review )

- add user-friendly font-size for  Maintenance and Support & Outstaffing blocks

It would be better to have work with designer

### PR DESCRIPTION
**Pull request by Sky7hunder**
Close #128

### CODE REVIEW

File: app/assets/stylesheets/mobile.scss [hunk 1]
```diff
@@ -139,6 +139,9 @@ div, p {
 
 div {
   text-align: justify;
+  h1, h2, h3, h4, h5 {
```
Comment (galulex) [inline app/assets/stylesheets/mobile.scss]: should it be here?

File: config/locales/en.yml [hunk 2]
```diff
@@ -110,17 +110,19 @@ en:
...
-      What's necessary to develop and maintain web product.
-      The main features of dedicated team realization can be short <h4>product releases</h4>,
-      halving functional step-by-step; there is no psychological problem as hiding troubles
-      from customers when product realization is; the customer has ability quickly responds
-      and makes decisions relating to functional improvement.
-      Ordering-company no needs to create a development and product testing division.
-      Save your financial and material resources, time and financial costs for the set
-      of a <h4>dedicated team</h4>; providing workplaces; training for the officers; utility costs.
+    team:
+      title: <h3>Web support</h3>
```
Comment (galulex) [inline config/locales/en.yml]: move html tag from here to slim

File: app/assets/stylesheets/mobile.scss (hunk 1 above)
Comment (Sky7hunder) [inline app/assets/stylesheets/mobile.scss]: looks like it works fine, but probably it needs some refactor

**Comment by Sky7hunder** [comment]
Large screen

Notebook screen

**Pull request merged**